subsys/bluetooth/adv_prov/providers/Kconfig.fast_pair

```
@@ -26,6 +26,13 @@ config BT_ADV_PROV_FAST_PAIR_ADV_BUF_SIZE
 	  Size of the buffer must be big enough to fit Fast Pair advertising
 	  packet.
 
+config BT_ADV_PROV_FAST_PAIR_SHOW_UI_PAIRING
```

Consider switching to the following name if you decide to use the previous API name suggestion

```suggestion
config BT_ADV_PROV_FAST_PAIR_UI_ENABLE
```

---

Show/hide UI indication would also be separately defined for Battery Notification Fast Pair extension. That's why I would prefer to keep information that the UI indication is related to pairing.